### DISCUSSION TITLE
Which URL to put at FILE_STORAGE_S3_URL ??

### DISCUSSION THREAD

**Discussion by edersong** [IgnisDa/ryot#635] [Q&A] [answered]
Trying to configure S3 bucket for custom exercises.
Created account and bucket on the AWS, but I didn't find any hint explaining which address to put at FILE_STORAGE_S3_URL environment setting.
Could you help me on it?

**Comment by IgnisDa** [comment]
Try leaving it empty. If the imports and export page loads, then it should be fine.

**Reply by edersong** [reply]
Thank you for the reply.
It's still not permiting to upload customer image for a Custom exercise :-(

**Reply by IgnisDa** [reply]
There is an env var for s3 region too. Look for it in the configuration docs and set that to yours.

**Reply by edersong** [reply, marked as answer]
Thank you!
Found and will put here the structure for who will need it also:

`https://region.amazonaws.com/bucket-name/
`

Change **_region_** to your region and **_bucket-name_** with your bucket
